helm/ffc-pay-statement-generator/templates/config-map.yaml

```diff
@@ -26,5 +26,5 @@ data:
   AZURE_STORAGE_CONTAINER: {{ quote .Values.container.storageContainer }}
   AZURE_STORAGE_FOLDER: {{ quote .Values.container.storageFolder }}
   STATEMENT_VERSION: {{ quote .Values.container.statementVersion }}
-  STATEMENT_RECEIVER_ENDPOINT: {{ quote .Values.container.statementReceiverEndpoint }}
+  STATEMENT_RECEIVER_ENDPOINT: {{ .Values.container.statementReceiverEndpoint }}-{{ .Values.environmentCode }}.{{ .Values.ingress.server }}
```

Comment:
Does this value need to be quoted as it was before?

Reply:
No can't do that.  If we quoted that value then it would only quote the first template.  eg  {{ quote .Values.container.statementReceiverEndpoint }}-{{ quote  .Values.environmentCode }}.{{ quote .Values.ingress.server }}

would result in something like "https://my-endpoint"-"my-env"."my-server"

Which obvs isn't a real URL.  There may be a way to combine and quote through go templating, but we know this will work as is because we've done the same else where and also similar above in the Service Bus topics